<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_register">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телевона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_login">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pw">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pw2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтверждение пароля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generator">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать следующее число</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить последний символ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reg2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>5</width>
      <height>5</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
